src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -2325,6 +2333,111 @@ def load_arguments(self, _):
                    help='Space-separated additional endpoint(s) to perform the connectivity check.',
                    validator=validate_custom_endpoints)
 
+    # Reference: https://learn.microsoft.com/en-us/cli/azure/k8s-extension?view=azure-cli-latest
+    with self.argument_context('aks extension') as c:
```

Comment:
should this be `aks extension create`? I think yes?

Reply:
we discussed offline and determined this was correct